test/jit/test_torchbind.py

```diff
@@ -111,7 +111,7 @@ def foo():
         # Ensure we are creating the object and calling __init__
         # rather than calling the __init__wrapper nonsense
         fc = FileCheck().check('prim::CreateObject()')\
-                        .check('prim::CallMethod[name="__init__"]')
+                        .check('prim::CallMethod[name="__init____0"]')
```

Comment:
Will this be fragile? Maybe we can just check for 
```
.check('prim::CallMethod[name="__init__')
```

Reply:
I think the name of call method will still be __init____0 tho. Are you saying we should just check if this method name has a substring "__init__" ?

Reply:
Apparently there is no method on FileCheck that does substring check.

Reply:
FileCheck is regex-based. If you don't include the closing quotation mark and bracket, it should work.

Reply:
oh lol my bad

Reply:
done